bril-txt/briltxt.py

```diff
@@ -146,7 +148,10 @@ def bool(self, items):
             return False
 
     def paramtype(self, items):
-        return {items[0]: items[1]}
+        if len(items) == 2:
+            return {items[0]: items[1]}
```

Comment:
Ah, this is a little worrisome. It means it's not possible to have one-argument parameterized types. Like, doesn't `product<int>` now do something wrong (even if that's not a very practical type)? Not sure what to do about this, TBH… maybe the whole parameterized-type thing needs rethinking. 🙃